Code/GraphMol/DistGeomHelpers/catch_tests.cpp

```diff
@@ -607,7 +607,7 @@ TEST_CASE("double bond stereo not honored in conformer generator") {
     RWMol cp(*m);
     MolOps::addHs(cp);
     DGeomHelpers::EmbedParameters ps = DGeomHelpers::ETKDGv3;
-    ps.enforceChirality = false;
```

Comment:
why is this change in this PR?

Reply:
I have to make this change every time I make a branch, or the build does not work on ARM64.  This one slipped through and got committed.     It seems like the ps.enforceChirality =true; should be the right setting - should I make a separate PR for this?

Reply:
Got it, and after looking at the test itself I agree that the change makes sense.
No need for a separate PR for this.